pkg/git/repo_test.go

```diff
@@ -206,7 +206,7 @@ func Test_setGCAutoDetach(t *testing.T) {
 	}
 
 	getGCAutoDetach := func(ctx context.Context, repo *repo) (bool, error) {
-		cmd := exec.CommandContext(ctx, r.gitPath, "config", "--get", "gc.autoDetach")
+		cmd := exec.CommandContext(ctx, repo.gitPath, "config", "--get", "gc.autoDetach")
```

Comment:
@Warashi Sorry for the late comment, look like the `r` variable in the below line (L210) has the same issue with this fixed one. It works because both refer to the outer `r` variable 👀 Would be great if you could open another PR to fix this.

Reply:
I opened another PR to fix this with other improvements.
https://github.com/pipe-cd/pipecd/pull/4932